slack-notifier/README.md

```diff
@@ -15,7 +15,24 @@ Follow these steps to configure the webhook in Slack:
 
 4. Copy the webhook URL from the setup instructions and use it in the AWS Setup section that follows.
 
-### AWS Setup
+
+### AWS setup using CloudFormation 
+
+#### CloudFormation
+Choose **Launch Stack** to launch the AWS Health SMS Notifier template in the US East (N. Virginia) Region in your account:
+ 
+
+<a href="https://console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/new?stackName=SlackNotifier&templateURL=https://s3.amazonaws.com/aws-health-tools/Cloudformation-templates/slack-notifier.json" title="Launch Stack"><img src="launch_stack.png" alt="Launch Stack" /></a>
```

Comment:
Let's use the same image used for the 'SMS notifier' template (see that README file)